src/math/calculation.js

```diff
@@ -854,7 +854,8 @@ function hypot(x, y, z) {
 p5.prototype.fract = function(num) {
   p5._validateParameters('fract', arguments);
   let sign = 0;
-  if (num === undefined || Math.abs(num) === Infinity) {
+  num = Number(num);
```

Comment:
Can the casted value from `Number(num)` be assigned to a new variable instead of reusing `num`? Problem could arise if someone accidentally pass an Object into `fract` and this line will change the variable holding the object into `NaN`.

Reply:
I think it works now?